<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Сервер|</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>IP адрес:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serv_addr">
        <property name="text">
         <string>192.168.100.4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Порт</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="serv_port">
        <property name="text">
         <string>8000</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>connect/disconnect</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QTreeWidget" name="tree_dir">
        <attribute name="headerCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="headerDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="headerHighlightSections">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Директория</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Размер файла</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата последней модификации</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата создания</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="terminal"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="path">
        <property name="text">
         <string>D:\\dir</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="select_path_button">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="scan_dir_button">
      <property name="text">
       <string>scan</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="input_line"/>
    </item>
    <item>
     <widget class="QPushButton" name="send_button">
      <property name="text">
       <string>Ок</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
